File: Placement Sheet/Accenture Interview Questions/StringSort.java
import java.util.Arrays;
import java.util.Scanner;

public class StringSort {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the string: ");
        String str = input.next();
        System.out.println("Unsorted string: " + str);
        System.out.println("Sorted string: " + sortString(str));
        input.close();
    }

    private static String sortString(String str) {
        if (str == null || str.length() == 0) {
            return "";
        }
        char[] charArr = str.toCharArray();

        Arrays.sort(charArr);

        return new String(charArr);
    }
}

File: Placement Sheet/Accenture Interview Questions/StringSort.py
def sortString(str: str):
    if len(str) == 0:
        return ""
    
    sortedStr = ''.join(sorted(str))
    return sortedStr

str = (str)(input("Enter the string: "))

print(f"Unsorted string: {str}")

print(f"Sorted string: {sortString(str=str)}")